core/src/main/java/org/apache/iceberg/TableProperties.java

```diff
@@ -207,6 +207,15 @@ private TableProperties() {}
   public static final String SPLIT_SIZE = "read.split.target-size";
   public static final long SPLIT_SIZE_DEFAULT = 128 * 1024 * 1024; // 128 MB
 
+  public static final String ADAPTIVE_SPLIT_SIZE_ENABLED = "read.split.adaptive-size.enabled";
+  public static final boolean ADAPTIVE_SPLIT_SIZE_ENABLED_DEFAULT = false;
```

Comment:
I think it would be best to disable this initially cause it is such a sensitive place to change. If performs well, we may enable in the future.